.NET/Microsoft.Recognizers.Definitions.Common/English/DateTimeDefinitions.cs

```diff
@@ -328,10 +329,8 @@ public static class DateTimeDefinitions
         };
       public static readonly Dictionary<string, string> SpecialYearPrefixesMap = new Dictionary<string, string>
         {
-            { @"calendar", @" " },
```

Comment:
Why remove? My question was just why the value in the dictionary is " " and not "", for example.

Reply:
Because the result of "calendar year" is same as "year", different from "school year".  So I deleted it to avoid distinguishing between "calendar" and "special year" again, make the code clear.